playbooks/robusta_playbooks/job_actions.py

```diff
@@ -0,0 +1,92 @@
+from robusta.api import *
+
+
+class JobParams(ActionParams):
+    """
+    :var image: The job image.
+    :var command: The job command as array of strings
+    :var name: Custom name for the job and job container.
+    :var namespace: The created job namespace.
+    :var restart_policy: Job container restart policy
+    :var job_ttl_after_finished: Delete finished job ttl (seconds). If omitted, jobs will not be deleted automatically.
+    :var notify: Add a notification for creating the job.
+
+    :example command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
+    """
+    image: str
+    command: List[str]
+    name: str = "robusta-action-job"
+    namespace: str = "default"
+    restart_policy: str = "OnFailure"
+    job_ttl_after_finished: int = None
+    notify: bool = False
+
+
+@action
+def alert_handling_job(event: PrometheusKubernetesAlert, params: JobParams):
```

Comment:
Do you think we should also add a non-alert related job creation action?

Reply:
I wouldn't add it in advance.
If someone will ask, we can surely add it